{"success": false, "error": "Traceback (most recent call last):\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 544, in _run_solc\n    ret: Dict = json.loads(stdout)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/json/__init__.py\", line 357, in loads\n    return _default_decoder.decode(s)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/json/decoder.py\", line 337, in decode\n    obj, end = self.raw_decode(s, idx=_w(s, 0).end())\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/json/decoder.py\", line 355, in raw_decode\n    raise JSONDecodeError(\"Expecting value\", s, err.value) from None\njson.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/slither/__main__.py\", line 826, in main_impl\n    ) = process_all(filename, args, detector_classes, printer_classes)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/slither/__main__.py\", line 86, in process_all\n    compilations = compile_all(target, **vars(args))\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/crytic_compile.py\", line 637, in compile_all\n    compilations.append(CryticCompile(target, **kwargs))\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/crytic_compile.py\", line 117, in __init__\n    self._compile(**kwargs)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/crytic_compile.py\", line 548, in _compile\n    self._platform.compile(self, **kwargs)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 154, in compile\n    targets_json = _get_targets_json(compilation_unit, self._target, **kwargs)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 276, in _get_targets_json\n    return _run_solc(\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 548, in _run_solc\n    raise InvalidCompilation(f\"Invalid solc compilation {stderr}\")\ncrytic_compile.platform.exceptions.InvalidCompilation: Invalid solc compilation Error: Multiple SPDX license identifiers found in source file. Use \"AND\" or \"OR\" to combine multiple licenses. Please see https://spdx.org for more information.\n--> /Users/giacomoibba/Desktop/mainnet/00/0000000BE1d98523B5469AfF51A1e7b4891c6225_Platform.sol\n\nError: Source \"solmate/auth/Owned.sol\" not found: File not found. Searched the following locations: \"\".\n    --> /Users/giacomoibba/Desktop/mainnet/00/0000000BE1d98523B5469AfF51A1e7b4891c6225_Platform.sol:1439:1:\n     |\n1439 | import {Owned} from \"solmate/auth/Owned.sol\";\n     | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"solmate/tokens/ERC20.sol\" not found: File not found. Searched the following locations: \"\".\n    --> /Users/giacomoibba/Desktop/mainnet/00/0000000BE1d98523B5469AfF51A1e7b4891c6225_Platform.sol:1440:1:\n     |\n1440 | import {ERC20} from \"solmate/tokens/ERC20.sol\";\n     | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"solady/utils/SafeTransferLib.sol\" not found: File not found. Searched the following locations: \"\".\n    --> /Users/giacomoibba/Desktop/mainnet/00/0000000BE1d98523B5469AfF51A1e7b4891c6225_Platform.sol:1441:1:\n     |\n1441 | import {SafeTransferLib} from \"solady/utils/SafeTransferLib.sol\";\n     | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"solmate/utils/ReentrancyGuard.sol\" not found: File not found. Searched the following locations: \"\".\n    --> /Users/giacomoibba/Desktop/mainnet/00/0000000BE1d98523B5469AfF51A1e7b4891c6225_Platform.sol:1442:1:\n     |\n1442 | import {ReentrancyGuard} from \"solmate/utils/ReentrancyGuard.sol\";\n     | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"src/interfaces/GaugeController.sol\" not found: File not found. Searched the following locations: \"\".\n    --> /Users/giacomoibba/Desktop/mainnet/00/0000000BE1d98523B5469AfF51A1e7b4891c6225_Platform.sol:1443:1:\n     |\n1443 | import {GaugeController} from \"src/interfaces/GaugeController.sol\";\n     | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"solady/utils/FixedPointMathLib.sol\" not found: File not found. Searched the following locations: \"\".\n    --> /Users/giacomoibba/Desktop/mainnet/00/0000000BE1d98523B5469AfF51A1e7b4891c6225_Platform.sol:1444:1:\n     |\n1444 | import {FixedPointMathLib} from \"solady/utils/FixedPointMathLib.sol\";\n     | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\n\n", "results": {}}
